server/game/TargetInterfaces.ts

```diff
@@ -12,11 +12,16 @@ import { PlayerTargetSystem } from './core/gameSystem/PlayerTargetSystem';
 export type ITriggeredAbilityTargetResolver<TContext extends TriggeredAbilityContext = TriggeredAbilityContext> =
   | ICardTargetResolver<TContext>
   | ISelectTargetResolver<TContext>
+  | ISelectStringTargetResolver<TContext>
```

Comment:
The existing Select mode is already selecting from a set of strings when you think about it. More descriptive names might include SelectButton and SelectDropdown, if I'm understanding the intended UI

Reply:
Yeah good callout, I wrestled with this one a bit. I was trying to avoid anything that describes exactly how the FE should work for the sake of decoupling, but I guess we are explicitly providing buttons and stuff now that I think about it. So I'll rename this to mention dropdown. Thanks!